iblvideo/run.py

```diff
@@ -19,6 +19,8 @@
                -2: 'Waiting',
                -3: 'Incomplete'}
 
+# TODO: make compatible with Lightning Pose
```

Comment:
This code can be simplified a lot as the task now handles the downloading of the data and registering of the data. Not a priority for now but when you implement this we can simplify both parts of the code